radius: 25.0 # 轨迹圆弧半径
cy: 70.0 # 轨迹直线长度
traj_length: 20 # 发布轨迹的长度
timer_period: 0.05 # 控制器周期
max_velocity: 12.0 # 最大速度
max_steer_rate: 0.3 # 最大转向速率
mpc_steps: 6 # mpc 预测布数
mpc_dt: 0.15 # mpc 周期
kcte: 1.1 # 四个权重
kepsi: 2.0
kev: 0.01
ksteer_cost: 5.0